src/main/java/com/afornalik/ox/controller/SkirmishController.java

```diff
@@ -0,0 +1,58 @@
+package com.afornalik.ox.controller;
+
+import com.afornalik.ox.model.board.Board;
+import com.afornalik.ox.model.board.FieldStatus;
+import com.afornalik.ox.model.board.OutOfBoardException;
+import com.afornalik.ox.model.player.PlayerContainer;
+import com.afornalik.ox.view.UIOperations;
+
+class SkirmishController {
```

Comment:
I'd be honest. I had to translate word "skrimish". Maybe use some simple words, like play, round or game instead? I don't think skirmish is widely used word.

Reply:
Changed to MatchController